doc_source/resource-type-walkthrough.md

```diff
@@ -4,9 +4,9 @@ In this walkthrough, we'll use the CloudFormation CLI to create a sample resourc
 
 ## Prerequisites<a name="resource-type-walkthrough-prereqs"></a>
 
-For purposes of this walkthrough, it's assumed you have already set up the CloudFormation CLI and associated tooling for your Java development environment:
-
-[Setting up your environment for developing extensions](what-is-cloudformation-cli.md#resource-type-setup)
+- Java 8
```

Comment:
```suggestion
- For this walkthrough, you'll need Java 8. You can also choose to use Java 11 instead, provided that you make the following changes to files you'll use later on as part of this example:
  - you'll need to change `"runtime": "java8",` to `"runtime": "java11",` in the `.rpdk.config` file;
  - you'll need to change both occurrences of `Runtime: java8` to `Runtime: java11` in the `template.yml` file;
  - you'll need to comment out both the `<maven.compiler.source>1.8</maven.compiler.source>` and `<maven.compiler.target>1.8</maven.compiler.target>` lines in the `pom.xml` file, and  add a `<release>11</release>` line inside the `<configuration>` block for the [maven-compiler-plugin](https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html) in the same `pom.xml` file.
```

Reply:
Hi @mrog - I just edited my suggestion above for you to review/add, because I realized that I omitted to escape the two dots (after `Java 8` on line `7`, and at the end of line `10`), that you can now see escaped in my suggestion, for consistency.  Please review/add the suggestion - thank you!